sway-lib-std/src/constants.sw

```diff
@@ -1,7 +1,8 @@
 //! Base asset and zero address constants.
 library;
 
-use ::contract_id::ContractId;
```

Comment:
It doesn't look like this import was used.

Reply:
Uh yeah thats why i removed that line